README.md

```diff
@@ -15,8 +15,19 @@
     </a>
 </p>
 
-`connqc` is a tool to control the quality of the network transport between multiple regions.
-It consists of a server that listens for TCP probing messages, and a client that sends said messages.
+`connqc` is a tool to check the quality of the network transport between networked hosts.
+We have, in our past encountered the following problems in the wild between two hosts:
+- The network connection between two hosts is not stable.
+- The network connection between two hosts may seem stable, but the firewall in between may be dropping packets.
+- The network connection between two hosts may seem stable, but the firewall in between may be dropping packets after a certain amount of time.
+- The network connection between two hosts may seem stable, but the firewall in between may be dropping packets after a certain amount of time, but only if the connection is established for a while.
+- The network connection between two hosts may seem stable, but certain ports or port combinations are blocked.
+- The network connection seems stable, but the routes taken are flipping and thus creating spontaneous connection issues.
+- Some other weird stuff.
```

Comment:
WDYT about highlighting the core problem in each of these items. As most of them start with `The network connection between two hosts may seem stable`, it makes the list seem longer than it is.

Perhaps something along the lines of:
* Unstable connection between hosts
* Firewall dropping packets intermittently
* Ports or port combinations are blocked or become blocked
* Routes flip flop causing spontaneous connection issues

Or something like that.